VideoGenerator/benchmarkOutput.txt

```diff
@@ -0,0 +1,67 @@
+System:
```

Comment:
Could you link this file in the README and the wiki, so that people will find it quicker?